src/main/java/org/pesmypetcare/webservice/service/usermanager/UserServiceImpl.java

```diff
@@ -16,33 +18,28 @@
     @Autowired
     private UserDao userDao;
 
-    @Override
-    public void createUserAuth(UserEntity user, String password) throws FirebaseAuthException {
-        userDao.createUserAuth(user, password);
-    }
-
     @Override
     public void createUser(String uid, UserEntity userEntity) throws DatabaseAccessException, FirebaseAuthException {
         userDao.createUser(uid, userEntity);
     }
 
     @Override
-    public void deleteFromDatabase(String uid) throws DatabaseAccessException, DocumentException {
+    public void deleteFromDatabase(String token, String uid) throws DatabaseAccessException, DocumentException {
         userDao.deleteFromDatabase(uid);
     }
 
     @Override
-    public void deleteById(String uid) throws DatabaseAccessException, FirebaseAuthException, DocumentException {
+    public void deleteById(String token, String uid) throws DatabaseAccessException, FirebaseAuthException, DocumentException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 120 characters (found 127).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5492924)